lib/aca_entities/families/family_member.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Families
+    class FamilyMember < Dry::Struct
+      # TODO: Person hbx_id until we add hbx_id on family member
+      attribute :hbx_id,                                      Types::String.optional.meta(omittable: false)
+      attribute :is_primary_applicant,                        Types::Bool.optional.meta(omittable: false)
+      attribute :is_consent_applicant,                        Types::Bool.optional.meta(omittable: false)
+      attribute :is_coverage_applicant,                       Types::Bool.optional.meta(omittable: false)
+      attribute :is_active,                                   Types::Bool.optional.meta(omittable: true)
+      attribute :applicant_reference,                         AcaEntities::MagiMedicaid::ApplicantReference.optional.meta(omittable: true)
+      attribute :former_family_reference,                     AcaEntities::Families::FamilyReference.optional.meta(omittable: true)
```

Comment:
A database query (scope) can identify the Families in which a Person is. Family Member.  A Person may belong to many family members, but may be the primary family member on only one.

Remove former_family_reference attribute until business case is clear.  Once business case is presented and it indicates need for this attribute (which i doubt will be the proven) model appropriately to reflect that a family_member may be present in more than one 'former' family

Reply:
Removed former_family_reference attribute and will add back after verifying business model